### ISSUE TITLE
Handling SessionEndedRequest

### ISSUE THREAD

**Issue by jishnu7** [johnwheeler/flask-ask#26]
`session_ended` is handled in sample code by returning `('', 200)`.

It will cause an error and alexa will say "There is a problem in communicating with the requested skill".

This can be solved by changing the code to 
``` 
 @ask.session_ended
 def session_ended():
     return statement("")
```
but the property `shouldEndSession: true` in `statement` is unnecessary here.

Why not flask-ask itself handle the response for `SessionEndedRequest`. In that case, flask-ask will be able to send a `_Response`, without any additional property. 

I can send a PR, if you think this method is fine.

**Comment by johnwheeler** [comment]
Are you sure it is not a problem with your code? I've never experienced this problem and the documentation explicitly states SessionEndedRequest cannot return a response

On this page:

https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/custom-standard-request-types-reference#sessionendedrequest

It says:

Valid Response Types

Your skill cannot return a response to SessionEndedRequest

**Comment by jishnu7** [comment]
I'm wasn't doing anything particular in session_end. 

```
@ask.session_ended
def session_ended():
    return '', 200
```

Will verify it again and let you know.

**Comment by relighted** [comment]
It indeed only works right when you end with an empty statement "". Skill will not be approved either if it ends in an error.

**Comment by fotoetienne** [comment]
I ran into this issue as well. My skill was receiving the error, "There is a problem in communicating with the requested skill". Returning an empty statement resolved the issue:

     @ask.session_ended
     def session_ended():
         return statement("")

**Comment by johnwheeler** [comment]
@fotoetienne You cannot return a statement on session_ended.

On this page:

https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/custom-standard-request-types-reference#sessionendedrequest

It says:

Valid Response Types

Your skill cannot return a response to SessionEndedRequest

**Comment by relighted** [comment]
@johnwheeler, no point arguing on the documentation ; but when  you do as the examples state,  and use the utterance 'quit', the Alexa response will be that there was an issue communicating with the skill.  The 'quit'  utterance is a default test for skill publishing -  if it returns an error the skill will fail certification.

**Comment by strunker** [comment]
I too actually just started a new thread on this.

return statement("") solved my issue with the error

**Comment by hpduong** [comment]
I was having the same issue with errors using the original code, too.

```
@ask.session_ended
def session_ended():
    return '', 200

```
Changing to @jishnu7 's code fixed it for me. 
```
 @ask.session_ended
 def session_ended():
     return statement("")
```

**Comment by pzp1997** [comment]
I was having the same issue when a question would "time out." I found the solution by looking at the samples, which have been updated to fix this problem (see ba020d1). The correct implementation for `session_ended` is

```
@ask.session_ended
def session_ended():
    return '{}', 200
```

It seems that ASK expects the response to be a valid JSON-encoded string, which the empty string is not.
